examples/nodejs/testingbot/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "stressbot",
+  "version": "1.0.0",
+  "description": "HiFi-JS allows developers to integrate High Fidelity's spatial audio technology into their JavaScript projects. This is a NodeJS example application that implements the High Fidelity Audio API.",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "bots": "node ./integration-test.js",
+    "bandwidth": "node --unhandled-rejections=strict -- ./testBandwidth.js"

```

With the latest client library, you'll need:

```suggestion
    "bots": "node -r esm ./integration-test.js",
    "bandwidth": "node -r esm --unhandled-rejections=strict -- ./testBandwidth.js"
```